improvelib/utils.py

```diff
@@ -480,7 +480,6 @@ def store_predictions_df(params: Dict,
                          y_true: np.array,
                          y_pred: np.array,
                          stage: str,
-                         outdir: Union[Path, str],
                          round_decimals: int = 4):
```

Comment:
I prefer to keep outdir and to get rid of params. We should make the calls as explicit as possible.

Reply:
Doesn't really matter to me, just seems easier than passing multiple values. @adpartin thoughts?